scio-avro/src/test/scala/com/spotify/scio/avro/types/TypeProviderTest.scala

```diff
@@ -30,7 +30,7 @@ import scala.reflect.runtime.universe._
 
 class TypeProviderTest extends AnyFlatSpec with Matchers {
   @AvroType.fromSchema(
-    """{"type":"record","name": "Record","fields":[{"name":"f1","type":"int"}]}"""
+    "{\"type\":\"record\",\"name\": \"Record\",\"fields\":[{\"name\":\"f1\",\"type\":\"int\"}]}"
```

Comment:
This is not part of `SortModifiers`? Is it from a new scalafmt version? Feels less readable than the triple-double-quoted version.

Reply:
yeah I saw this as well and this is caused by the SortModifiers only on annotations. I haven't had time yet to track those down and change them to:

```scala
  @AvroType.fromSchema("""
      |{"type":"record","name": "Record","fields":[{"name":"f1","type":"int"}]}
      |""".stripMargin)
  class StringLiteralRecord
```